assets/js/theme/global/currency-selector.js

```diff
@@ -1,5 +1,57 @@
-export default function () {
+import swal from './sweet-alert';
+import utils from '@bigcommerce/stencil-utils';
+
+export default function (cartId) {
+    function changeCurrency(url, currencyCode) {
+        $.ajax({
+            url,
+            method: 'POST',
```

Comment:
I think by default, the `content-type` for Ajax requests made with jQuery is `application/x-www-form-urlencoded`. Considering you're sending a JSON body back, it'd be good if you can pass `contentType: 'application/json'`.